examples/stress_tests/README.md

```diff
@@ -1,3 +1,10 @@
 # Stress tests
 
-These examples are used to stress test Bevy's performance in various ways. These should be run with the --release argument to cargo or equivalent optimization, otherwise they will be very slow.
+These examples are used to stress test Bevy's performance in various ways. 
+These should be run with the "stress-test" profile, otherwise they will be very slow or inaccurately represent production level performance.
+
+## Example Command
+
+```
+cargo run --example <EXAMPLE> --profile stress-test
```

Comment:
```suggestion
cargo run --profile stress-test --example <EXAMPLE> 
```
just so that it's nicer to copy-paste in a terminal and change the example name